build.cmd

```diff
@@ -1,3 +1,3 @@
 @echo off
```

Comment:
Does `clean.cmd` still work as is or do we need to update it?
https://github.com/dotnet/corefxlab/blob/154fafa095dd6159867c4aeb096d31c60ae96ad0/clean.cmd#L6-L11

Reply:
We should remove `clean.cmd` and people should just use build -clean... which will remove the artifacts directory.

Reply:
Sounds good. Let's remove it.

I will copy this script to my dev box instead, since I use it to close all dotnet.exe instances running (I don't think `build -clean` does that).

Reply:
We can of course propose that into arcade... it would be handy if arcade searched for dotnet.exe and killed it.